packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -834,18 +835,19 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
   final CapturedThemes capturedThemes;
   final BoxConstraints? constraints;
   final Clip clipBehavior;
+  final AnimationStyle? popUpAnimationStyle;
 
   @override
   Animation<double> createAnimation() {
```

Comment:
Similar to MaterialApp: if popupAnimationStyle is noAnimation, can we leave the CurvedAnimation out?